lib/ios/native-navigation/Dictionary+FunctionalExtensions.swift

```diff
@@ -40,12 +40,14 @@ extension Dictionary {
     return dict
   }
 
-
+#if swift(>=3.2)
```

Comment:
If it is Swift 4 only why we don't add `mapValues` in Swift 3.2?

Reply:
After trying this fix for my a clean project, I am getting an error `Method does not override any method from its superclass` for `constantsToExport` in `SharedElementGroupManager.swift`.